### ISSUE TITLE
Add input/outputShape information to all KDocs of layers

### ISSUE THREAD

**Issue by zaleslaw** [Kotlin/kotlindl#211]
We need to have better documentation for our layers.
A few layers have inputShape/outputShape information in the KDocs, copied from Keras documentation, for example 
[GlobalAvgPool2D](https://github.com/JetBrains/KotlinDL/blob/41f0b20d3bfd6e277d8486d945cdef59a6250b9b/api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/pooling/GlobalAvgPool2D.kt#L20)

Please add the same documentation section to all the following layers:

- [ ] ELU
- [ ] LeakyReLU
- [ ] PReLU
- [ ] ReLU
- [ ] Softmax
- [ ] ThresholdedReLU
- [ ] DepthwiseConv2D
- [ ] SeparableConv2D
- [ ] Dense
- [ ] Input
- [ ] BatchNorm
- [ ] AvgPool1D
- [ ] AvgPool2D (should be in one format)
- [ ] AvgPool3D
- [ ] GlobalMaxPool1D
- [ ] GlobalMaxPool2D
- [ ] GlobalMaxPool3D
- [ ] MaxPool1D
- [ ] MaxPool2D
- [ ] MaxPool3D
- [ ] Flatten
- [ ] ZeroPadding1D
- [ ] ZeroPadding2D
- [ ] ZeroPadding3D

**Comment by dosier** [comment]
Hey, I'd like to contribute a bit again. Could do this one. 

One question I have, in some cases Keras specifies different shapes depending upon the `data_format` being `channels_first` or `channels_last`.  I do see we have this implemented for the `ZeroPadding..` layers but not for any others. 

From what I understand the default implementation uses the `channels_last` format.

> There are two conventions for shapes of images tensors: the channels-last convention (used by TensorFlow) > and the channels-first convention (used by Theano).

 **Deep Learning with Python - François Chollet**
 (https://stackoverflow.com/a/54967450/12089702)

So I guess for now we just omit the part about the `data_format` and only take the `channels_last` part?
